packages/core/admin/server/services/user.js

```diff
@@ -257,6 +273,11 @@ const deleteByIds = async (ids) => {
     deletedUsers.push(deletedUser);
   }
 
+  strapi.eventHub.emit(
+    'user.delete',
+    deletedUsers.map((deletedUser) => sanitizeUser(deletedUser))
```

Comment:
this means we'd save an array as the json payload, I think it's allowed but I'd find it more readable if it was an array with a users key

Reply:
oh yeah good idea, I'll update